invenio_rdm_records/resources/serializers/dublincore/__init__.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2021 CERN.
```

Comment:
Hmm not sure if CERN is the copyright for your institution?

Reply:
The correct header to add is ``# Copyright (C) 2021 Graz University of Technology.`` (this is the on user other places).